_tests/build

```diff
@@ -24,6 +24,7 @@ else
 fi
 
 # Run HTMLProofer
+# Swap https://mtlynch.io/ for / to make URLs relative vs. absolute.
```

Comment:
One thing I realized from #331 is that it's easy to change the build script to solve a problem, but then later forget what problem it solved. I'll merge this in because I'd like to get the book reports up, but I'm just going to expand this comment a bit to explain our reasoning for this flag.